impit-node/.npmignore

```diff
@@ -11,3 +12,6 @@ yarn.lock
 .yarn
 __test__
 renovate.json
+*.rs
+test
+.yarnrc.yml
```

Comment:
time to add editor config so we always have the blank lines at the end? :]

Reply:
Copied over the one from `crawlee-python` (and installed the plugin into my VSCode finally 🙃 ). Is there a way to format the existing files to conform to the `.editorconfig` now (e.g. indent space count)?

Reply:
Biome or prettier I guess?

Reply:
Both of those seem to be primarily targetting JavaScript. Since all of the JavaScript code we now have is automatically generated, there is imo not much point in that right now.

I already enabled `cargo format` and `clippy` for the rust part of the codebase, let's add more (JS / Python) formatters once we write more of those languages, I guess.